jujugui/static/gui/src/app/components/entity-details/header/header.js

```diff
@@ -46,8 +46,17 @@ class EntityHeader extends React.Component {
       // format the method expects.
       this.props.deployService(entityModel, undefined, plans, activePlan);
     } else {
-      var id = entity.id.replace('cs:', '');
-      this.props.getBundleYAML(id, this._getBundleYAMLCallback.bind(this));
+      const id = entity.id.replace('cs:', '');
```

Comment:
No need for this, just pass entity.id to jujulib.URL.fromString.

Reply:
Ah ok, I thought it was still needed for the getBundleYAML call, but I see below you can use bundleUrl.legacyPath() 👍